packages/compass/src/main/handle-exception.ts

```diff
@@ -45,4 +43,17 @@ async function handleUncaughtException(err: Error): Promise<void> {
   await showErrorMessageBox();
 }
 
-export { handleUncaughtException };
+
+function handleUncaughtException(err: Error): Promise<void> {
+  // eslint-disable-next-line no-console
+  console.error('handling uncaughtException', err);
```

Comment:
Are those left here by mistake? There is already a `console.error` in the handler method

Reply:
We should have some way of distinguishing the two, I guess I’d rather remove this from the handler method than here?

Reply:
To differentiate which error handling - rejection or exception. I wanted to preserve the existing behaviour here

Reply:
If we want to distinguish between the two then it should be part of the message we show in the dialog too I think, this is what users will share with us

Reply:
added in [1e30608](https://github.com/mongodb-js/compass/pull/4917/commits/1e30608896d5aeeabce1348eea2478ded9609955)